docs/authorization.md

```diff
@@ -89,28 +89,31 @@ Auth checker can be also defined as a class - this way we can leverage the depen
 
 ```ts
 export class CustomAuthChecker implements AuthCheckerInterface<ContextType> {
-  // inject dependency
-  constructor(private readonly userRepository: Repository<User>) {}
+  // Inject dependency
+  constructor(
+    @Inject()
```

Comment:
Is this a hypothetical `@Inject()` decorator to show the intention?
Now it's misleading that type-graphql exports `@Inject()` decorator.